tools/pythonpkg/src/python_objects.cpp

```diff
@@ -403,9 +403,9 @@ py::object PythonObject::FromValue(const Value &val, const LogicalType &type) {
 	}
 	case LogicalTypeId::INTERVAL: {
 		auto interval_value = val.GetValueUnsafe<interval_t>();
-		uint64_t days = duckdb::Interval::DAYS_PER_MONTH * interval_value.months + interval_value.days;
-		return py::cast<py::object>(import_cache.datetime().timedelta()(
-		    py::arg("days") = days, py::arg("microseconds") = interval_value.micros));
+		return py::cast<py::object>(import_cache.pandas().DateOffset()(
+		    py::arg("months") = interval_value.months, py::arg("days") = interval_value.days,
+		    py::arg("nanoseconds") = interval_value.micros * Interval::MICROS_PER_MSEC));
```

Comment:
This creates a dependency on Pandas in our DuckDB -> Python code in a path unrelated to Pandas.
Why is timedelta insufficient here?

Reply:
@Mause maybe you have a suggestion for an interval representation that does not add a dependency?

Reply:
Here's a solution that doesn't require going outside python stdlib: https://stackoverflow.com/a/4131114/651174. Example solution copied below:

```
import datetime, calendar

def add_months(sourcedate, months):
    month = sourcedate.month - 1 + months
    year = sourcedate.year + month // 12
    month = month % 12 + 1
    day = min(sourcedate.day, calendar.monthrange(year,month)[1])
    return datetime.date(year, month, day)

somedate = datetime.date.today()
add_months(somedate,1)
# datetime.date(2023, 4, 3)
```

Reply:
> @Mause maybe you have a suggestion for an interval representation that does not add a dependency?

I'm not aware of a standard library one aside from timedelta, but I see no issue with us defining our own? Either as named tuple or a dataclass